instrumentation-core/servlet/servlet.gradle

```
@@ -1,14 +1,10 @@
-group = 'io.opentelemetry.instrumentation'
-
-apply from: "$rootDir/gradle/java.gradle"
-apply from: "$rootDir/gradle/publish.gradle"
+ext {
+  javaSubPackage = ''
```

this is used by servlet auto-instrumentation. and i think you'll also need to update those auto-instrumentation to use `configuration: 'shadow'`, and use shaded package name
```suggestion
  javaSubPackage = 'servlet'
```